Dockerfile

```diff
@@ -1,12 +1,24 @@
 FROM gitlab-registry.cern.ch/swan/docker-images/dev-jupyter
+
+LABEL maintainer="Akash Ravi <akashkravi@gmail.com>"
+
 USER root
+
 RUN git clone https://github.com/techtocore/jupyter.git
 RUN cd jupyter && \
 rm -rf /opt/conda/lib/python3.7/site-packages/notebook/templates && \
 cp -rl templates /opt/conda/lib/python3.7/site-packages/notebook/ && \
 ls && \
 cd SwanContents && pip install --no-deps . && \
 jupyter nbextension install --py --system swancontents
+
+RUN git clone https://github.com/Anaconda-Platform/nb_conda_kernels.git
+RUN cd nb_conda_kernels && \
+cat requirements.txt | xargs -n 1 pip install || true && \
```

Comment:
`pip install -r requirements.txt`

Reply:
Can't you just `conda/pip install nb_conda_kernels`?

Reply:
> `pip install -r requirements.txt`

This is what I had in the beginning. Unfortunately, this breaks even if any one of the packages fails to install. I had a situation where it caused an issue. However, if the current version works, I'll revert it back to  `pip install -r requirements.txt`